src/testUtils/java/uk/gov/hmcts/reform/wataskmanagementapi/utils/Assertions.java

```diff
@@ -65,6 +71,57 @@ public void taskStateWasUpdatedInDatabase(String taskId, String value, Headers a
             .log();
     }
 
+    public void taskValuesWasUpdatedInDatabase(String taskId, Map<String, Matcher<?>> fieldValueMap,
+                                               Headers authenticationHeaders) {
+
+        Response result = restApiActions.get(
+            "task/{task-id}",
+            taskId,
+            authenticationHeaders
+        );
+
+        result.then().assertThat()
+            .statusCode(HttpStatus.OK.value())
+            .and().contentType(MediaType.APPLICATION_JSON_VALUE).log();
+
+        fieldValueMap.entrySet().forEach(
+            entry -> result.then().assertThat()
+                .body(entry.getKey(), entry.getValue()).log()
+        );
+    }
+
+    public void taskActionAttributesUpdatedInDatabase(String taskId, String assigneeId,
```

Comment:
Should this be in this class called `Assertions`?  Doesn't assert anything.  Seems to prepare a list of attributes to assert later which are passed in as individual params.

Reply:
just for reusing across different functional tests, i have added it here, currently its been used only by postCompletion functional test, later this method will be used in all other functional tests, can be used by others in future